docs: updates to dashboards descriptions and grafana tasks (#5591)
strimzi/strimzi-kafka-operator: documentation/modules/metrics/archive_metrics-dashboard-descriptions.adoc

// This assembly is included in the following assemblies:
//
// metrics/assembly-metrics-config-files.adoc

[id='ref-metrics-dashboards-{context}']
= Example Grafana dashboards

[role="_abstract"]
If you deploy Prometheus to provide metrics,

> you can use Strimzi's example Grafana dashboards to monitor Strimzi components.

Reviewer: Again, I wonder if we should be a bit more open to other tools. Our examples use Prometheus and Grafana, but other compatible tools can be used as well.

Author: I added a note, but this file is archived and not included in the doc. 
This files isn't used now.

> Example dashboards are provided in the `examples/metrics/grafana-dashboards` directory as JSON files.

Reviewer: ```suggestion
Example dashboards are provided in the `examples/metrics/grafana-dashboards` directory as JSON files in our link:https://github.com/strimzi/strimzi-kafka-operator/[GitHub repository^] or in the release archives.
```

> All dashboards provide JVM metrics, as well as metrics specific to the component.

Reviewer: Not everything is Java based, so not everything has JVM metrics.

```suggestion
The dashboards provide metrics specific to each component.
```

For example, the Grafana dashboard for Strimzi operators provides information on the number of reconciliations or custom resources they are processing.

The example dashboards don't show all the metrics supported by Kafka.

> The dashboards are populated with a representative set of metrics for monitoring.

Reviewer: I wonder if we have some better word than representative. The dashboards have the metrics which we think are important for monitoring. But it is always opinionated, so other people might see other metrics as important. The word `representative` is ok, but at least for me it does not have exactly the same meaning.

Author: I changed to _The dashboards are populated with a limited set of metrics that are key for monitoring._ But we don't include this file now. I kept it in case we want to reintroduce.
